5.0.0/agent_install.md

```diff
@@ -0,0 +1,24 @@
+# Agent Install
```

Comment:
Lets call this as **deprecared**

Reply:
We dont want to support agent Installs from Installer anymore.

Reply:
ok done